packages/connectors/connector-helper/package.json

```diff
@@ -27,7 +27,8 @@
     "@rarible/sdk-common": "^0.13.69-beta.1",
     "@rarible/sdk-wallet": "^0.13.69-beta.1",
     "@rarible/tezos-sdk": "0.1.46",
-    "@rarible/web3-ethereum": "^0.13.69-beta.1"
+    "@rarible/web3-ethereum": "^0.13.69-beta.1",
```

Comment:
Looks like it must be in peerDependencies (as optional as well) because user might want to use only one of them